README.md

```diff
@@ -1,2 +1,50 @@
 # LeaderBoard-API
 This is a leaderboard website that displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- webpack
+
+## Live Demo
+
+[Live Demo](https://direakanbi.github.io/LeaderBoard-API/)
+
```

Comment:
- [ ] The live demo currently opening as README. Please fix it and make sure the live link is working well to make the README professional.

Reply:
> * [ ]  The live demo currently opening as README. Please fix it and make sure the live link is working well to make the README professional.

Hello Mr Dipesh, given that I had consistent deployment errors that would be tended to in the nearest future, I removed the Live Demo from the README. Given that it is an optional feature, I will work on fixing it in due time. Thank you for your review 😄

Reply:
Hi @direakanbi.

The live demo link is an optional thing, but I suggested it because you have included it. If you are getting an error, for now, you can remove the `live demo link` from the README to make it professional.

Thank you
_**Dipesh Kumar**_